main.go

```diff
@@ -19,6 +19,12 @@ func main() {
 	// Set up Postgres connection
 	server := server.New(config)
 
+	// Start DB Schema check
+	if (!server.CheckDB()) {
+		// TODO: Do something when it fails
+		log.Fatal("RIP")
```

Comment:
`log.Fatal` is fine for now but please add a more descriptive error, ie "Database formatted incorrectly"